<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slicepod</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../slicepod.qrc">
    <normaloff>:/images/icon-128.png</normaloff>:/images/icon-128.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="upperPlayerLayout">
      <item>
       <layout class="QHBoxLayout" name="playerButtonsLayout">
        <item>
         <widget class="QToolButton" name="playButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../slicepod.qrc">
            <normaloff>:/images/media-playback-start.png</normaloff>:/images/media-playback-start.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="stopButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../slicepod.qrc">
            <normaloff>:/images/media-playback-stop.png</normaloff>:/images/media-playback-stop.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="addFragmentButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../slicepod.qrc">
            <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="editFragmentButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../slicepod.qrc">
            <normaloff>:/images/document-edit.png</normaloff>:/images/document-edit.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="playingStatusStaticLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Now playing:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="playingStatusLabel">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="PositionWidget" name="positionWidget" native="true"/>
    </item>
    <item>
     <widget class="FragmentEditorWidget" name="fragmentEditorWidget" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="libraryView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLibrary">
    <property name="title">
     <string>&amp;Library</string>
    </property>
    <addaction name="actionAdd_directory"/>
   </widget>
   <addaction name="menuLibrary"/>
  </widget>
  <action name="actionAdd_directory">
   <property name="text">
    <string>&amp;Add directory...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PositionWidget</class>
   <extends>QWidget</extends>
   <header>gui/positionwidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FragmentEditorWidget</class>
   <extends>QWidget</extends>
   <header>src/gui/fragmenteditorwidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../slicepod.qrc"/>
 </resources>
 <connections/>
</ui>
